pkg/kubebench/scanner.go

```diff
@@ -33,7 +33,7 @@ const (
 )
 
 type Config interface {
-	GetKubeBenchImageRef() string
```

Comment:
Don't break this interface. See my comment about SOLID and interface segregation.